src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerDao {
+    Manufacturer create(Manufacturer manufacturer);
+
+    Manufacturer get(Long id);
```

Comment:
It's better to use Optional in get method because it can return NullPointerException
```suggestion
    Optional<Manufacturer> get(Long id);
```